src/main/java/frc/robot/OI.java

```diff
@@ -1,8 +1,85 @@
 package frc.robot;
 
+import com.spikes2212.command.genericsubsystem.commands.MoveGenericSubsystem;
+import com.spikes2212.util.PlaystationControllerWrapper;
+import com.spikes2212.util.smartmotorcontrollers.SparkWrapper;
+import edu.wpi.first.wpilibj2.command.ConditionalCommand;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.commands.*;
+import frc.robot.subsystems.*;
+
 public class OI /*GEVALD*/{
 
+    private final PlaystationControllerWrapper driverJoystick = new PlaystationControllerWrapper(0);
+    private final PlaystationControllerWrapper navigatorJoystick = new PlaystationControllerWrapper(1);
+
+    private boolean inAlgaeMode;
+
     public OI() {
```

Comment:
instead of calling `getInstance()` everytime, store the value in a field and use that field